src/components/user/resources/Hackpack.tsx

```diff
@@ -14,31 +14,34 @@ const Hackpack = ({ text, techs, link, description }: props) => {
     <div className="rounded-xl bg-white p-3">
       <Link
         data-cy="hackpack-link"
-        className="flex items-center justify-between opacity-100 hover:opacity-70"
+        className="items-center justify-between opacity-100 hover:opacity-70"
         href={link}
         target="_black"
       >
-        <p className="text-lg font-semibold" data-cy="hackpack-text">
+        <div
+          className="flex items-center justify-between text-lg font-semibold"
+          data-cy="hackpack-text"
+        >
           {text}
-        </p>
-        <Github size={20} />
-      </Link>
-
-      <div className="my-2 flex flex-wrap gap-2" data-cy="hackpack-techs">
-        {techs.map((tech, index) => (
-          <div
-            key={index}
-            className="flex items-center gap-2 text-gray-400"
-            data-cy="hackpack-tech"
-          >
-            <div className="text-hackathon-blue-100" data-cy="hackpack-icon">
-              {TECHSTACKS[tech]}
-            </div>
-            {tech}
+          <div>
+            <Github size={20} />
           </div>
-        ))}
-      </div>
-
+        </div>
+        <div className="my-2 flex flex-wrap gap-2" data-cy="hackpack-techs">
+          {techs.map((tech, index) => (
+            <div
+              key={index}
+              className="flex items-center gap-2 text-gray-400"
+              data-cy="hackpack-tech"
+            >
+              <div className="text-hackathon-blue-100" data-cy="hackpack-icon">
+                {TECHSTACKS[tech]}
+              </div>
+              {tech}
+            </div>
+          ))}
+        </div>
+      </Link>
       <p className="mt-2">{description}</p>
```

Comment:
the entire card should be in the link component, so the description should also be included in the link